### ISSUE TITLE
Update Maven repository

### ISSUE THREAD

**Issue by Ohems** [jroyalty/jglm#15]
I see that the Mat4 x Vec4 multiplication was added to the library as far back as Jun 26, 2016. Do you have any plans to release these features to the Maven repository which is still lagging behind at 1.0.0?

**Comment by jroyalty** [comment]
Yeah, I can do another release.  I'll land #16 and do a release soon.

**Comment by Newbytee** [comment]
I know it's almost been 4 years, but what are the chances you could make that release actually happen?
